src/scripts/main.js

```diff
@@ -1,3 +1,24 @@
 'use strict';
 
-// write code here
+const treeItems = document.querySelectorAll('.tree li');
+
+treeItems.forEach((item) => {
+  const childUl = item.querySelector('ul');
+
+  if (childUl) {
+    const span = document.createElement('span');
+
+    span.textContent = item.firstChild.textContent;
```

Comment:
Accessing `item.firstChild.textContent` assumes that the first child is a text node. If the first child is not a text node, this will result in an error. Consider checking if `item.firstChild` is a text node or use a more reliable method to get the text content.